source/intel.js

```diff
@@ -2,11 +2,7 @@
 
 const sat_version = "2.21";
 
-const stripHtml = (html) => {
-  let tmp = document.createElement("DIV");
-  tmp.innerHTML = html;
-  return tmp.textContent || tmp.innerText || "";
-};
+import { stripHtml } from "./strip.js";
```

Comment:
We should remove this & find one without vulnerabilities.

Reply:
Or just stop using it, by just deleting its import and one usage case.

Reply:
I'll make a new PR with a different example to replace this one.